rn/Teacher/ios/TeacherUITests/TeacherUITestCase.swift

```diff
@@ -0,0 +1,35 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2018-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import Foundation
+import XCTest
+@testable import Core
+@testable import TestsFoundation
+
+class TeacherUITestCase: UITestCase {
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = false
+        if app.state != .runningForeground {
+            launch()
+        }
+        reset()
+        // mockDownload(URL(string: ".")!) // Use only mock data
```

Comment:
It would be nice if we end up writing all these tests with mock data, but Login tests probably need to be live.